clang/test/CodeGenSYCL/intel-fpga-mem-builtin.cpp

```diff
@@ -9,8 +9,12 @@ struct State {
   float y;
 };
 
```

Comment:
Please add a comment about what this test is doing.

Reply:
Done.